README.md

```diff
@@ -23,6 +23,7 @@ the direction the language is evolving in.
 * [0016-more-attributes.md](text/0016-more-attributes.md)
 * [0019-opt-in-builtin-traits.md](text/0019-opt-in-builtin-traits.md)
 * [0048-traits.md](text/0048-traits.md)
+* [0059-remove-tilde.md](text/0059-remove-tilde.md)
```

Comment:
@alexcrichton I'll let you rename [subtask of RFC 59: Implement associated constants](https://github.com/rust-lang/rust/issues/17841) to the correct RFC ^_^.